source/install/cloud-changelog.md

```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 - [Release 2022-03-02](#release-2022-03-02)
 - [Release 2022-02-16](#release-2022-02-16)
 - [Release 2022-02-10](#release-2022-02-10)
-- [Release 2022-02-03](#release-2022-02-03)
+
+## Release 2022-03-30
+
+### Improvements
+
+#### User Interface (UI)
+ - For Apps, calls are now separated between submit, form, refresh and lookup calls.
```

Comment:
Is there anything else that Cloud users should know about this? @larkox

Reply:
This is more intended for Apps developers than for users. The experience for users has not changed.

Reply:
The only potential change if they have already created their own apps, that may stop working, since they have to be updated to the new system, but since, I think, there are not many people creating apps out there apart of us, should be no problem.